be1-go/message/messagedata/reaction_add.go

```diff
@@ -10,7 +11,7 @@ type ReactionAdd struct {
 	Object            string `json:"object"`
 	Action            string `json:"action"`
 	ReactionCodepoint string `json:"reaction_codepoint"`
-	ChirpId           string `json:"chirp_id"`
+	ChirpID           string `json:"chirp_id"`
 	Timestamp         int64  `json:"timestamp"`
```

Comment:
I know It's not because of you, I think that "// Timestamp is a Unix timestamp " must be added before Unix times